pkg/nad/masterplugin.go

```diff
@@ -277,7 +277,7 @@ func (plugin *MasterVlanPlugin) WithLinkInContainer() *MasterVlanPlugin {
 		return plugin
 	}
 
-	plugin.masterPlugin.LinkInContainer = true
+	plugin.masterPlugin.LinksInContainer = true
```

Comment:
LinkInContainer and LinksInContainer are two different parameters. Please revert it back to it's original state